x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```diff
@@ -0,0 +1,55 @@
+.appSearchResult {
+  overflow: hidden;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
+    width: 100%;
+    padding: $euiSize;
+    display: grid;
+    grid-template-rows: 1fr $euiSizeXS * 5;
+    border-right: $euiBorderThin;
+  }
+
+  &__contentInner {
+    display: grid;
+    grid-gap: $euiSizeS;
+    overflow: hidden;
+  }
+
+  &__fieldsetContainer {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__hiddenFieldsIndicator {
+    font-size: $euiFontSizeXS;
+    color: $euiColorDarkShade;
+    align-self: end;
+    padding-top: 0;
+  }
+
+  &__toggleExpandButton {
+    justify-self: stretch;
+    align-self: stretch;
+    display: flex;
+    width: $euiSizeL * 2;
+
+    &:hover,
+    &:focus,
+    &:active {
+      background-color: $euiPageBackgroundColor;
+      cursor: pointer;
+    }
+  }
+
+  &__toggleExpandButton {
+    grid-column: 2;
+  }
+
+  &__toggleIcon {
+    align-self: center;
+    margin: auto;
+  }
+}
```

Comment:
IMO, we can/should remove this extra CSS class entirely by just setting flex centering on the parent container:

```css
.appSearchResult__toggleExpandButton {
  justify-content: center;
  align-items: center;
}
```

Reply:
I actually just noticed/tested that `justify-self: stretch;` and `align-self: stretch;` aren't really doing anything on `.appSearchResult__toggleExpandButton` - let's go ahead and remove those lines as well.

Reply:
I consolidated all my button-related feedback into a diff for easy reference/cherry-picking: https://github.com/constancecchen/kibana/commit/d46f65587d4ad77c0b671c186feead9a95a7a96a